Common/Source/Dialogs/dlgRadioSettings.cpp

```diff
@@ -239,20 +239,20 @@ TCHAR Name[250];
     {
         if( ActiveRadioIndex > RESWP_END)
         {
-          _stprintf(Name,_T("%s"),WayPointList[ActiveRadioIndex].Name);
+          _sntprintf(Name,DEVICE_NAME_LEN,_T("%s"),WayPointList[ActiveRadioIndex].Name);
```

Comment:
Name size is 250 and WAYPOINT::Name is 31 so :
 - previous _stprintf usage can't overflow  -> no bug
 - using _sntprinf with size less than 31 can truncate string -> bug on utf8 platform